src/main/java/net/glowstone/entity/projectile/GlowEnderPearl.java

```diff
@@ -40,33 +44,67 @@ public GlowEnderPearl(Location location, float speed) {
         super(location);
         setDrag(0.99, false);
         setDrag(0.99, true);
-        setHorizontalAirDrag(0.95);
-        setGravityAccel(new Vector(0,-0.06,0));
+        setHorizontalAirDrag(1);
+        setGravityAccel(new Vector(0, -0.03, 0));
```

Comment:
Can you put the Y acceleration (`-0.03`) in a named constant in the class? e.g. `VERTICAL_GRAVITY_ACCEL`